### ISSUE TITLE
When "Write a post" button is focused using keyboard it is not highlighted

### ISSUE THREAD

**Issue by Rafi993** [thepracticaldev/dev.to#7949]
**Describe the bug**
When you focus on "Write a post" button using keyboard no styling specific to :focus is applied and the default outline is removed too

**To Reproduce**
1. Reload dev.to
2. Keep pressing tab key till you reach "Write a post" button
3. You cant see if it is focused since no distinctive style is applied for focus

**Expected behavior**
outline, border or background color should be applied when using :focus so that it appears distinctively.
  
**Screenshots**
![focus_on_new_button](https://user-images.githubusercontent.com/8092120/82335166-95af5b00-9a06-11ea-95c4-8d2bccf1c77c.png)

In the above screenshot I applied :focus pseudo selector on "Write a post" button using dev tools but the same result appears when you navigate to that button using keyboard.

**Desktop (please complete the following information):**

- OS: Ubuntu 18.04
- Browser: Brave
- Version: 1.8.96 
- Chromium: 81.0.4044.138 (Official Build) (64-bit)

**Comment by Rafi993** [comment]
I thing this is the case for all buttons with similar styling (black background and white text)

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by TheRohitSharma** [comment]
I'd love to take this one.

**Comment by rhymes** [comment]
@TheRohitSharma please go ahead, I'm going to assign you the issue
